### PR TITLE
feat: add variant prop to Calendar component

### PR DESCRIPTION
**Pull request by wildergd**
fix: #1554 

## Changes proposed in this PR:
- add `variant` prop to Calendar component

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

**Comment by wildergd** [comment]
i have to check accesibility test because i see there are to many failing so probably it is not an issue

File: src/components/Calendar/singleCalendar.js [hunk 1]
```diff
@@ -0,0 +1,360 @@
+/* eslint-disable react/prop-types */
```
Comment (yvmunayev) [inline src/components/Calendar/singleCalendar.js:1]: same before

File: src/components/Calendar/styled/calendarsContainer.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        flex: 1;
+    }
+
+    & > div:nth-child(1) {
+        margin-right: 2rem;
+    }
+
+    & > div:nth-child(2) {
+        margin-left: 2rem;
+    }
```
Comment (yvmunayev) [inline src/components/Calendar/styled/calendarsContainer.js:19]: don't do interpolated, declare a styled component for each

File: src/components/Calendar/styled/controlsContainer.js [hunk 3]
```diff
@@ -5,6 +5,22 @@ const StyledControlsContainer = styled.div`
...
+        flex-direction: column;
+
+        & > div:nth-child(1) {
+            order: 1;
+            width: 100%;
+        }
+
+        & > div:nth-child(2) {
+            padding: 0 8px 8px 0;
+        }
```
Comment (yvmunayev) [inline src/components/Calendar/styled/controlsContainer.js:22]: same before

**Pull request closed without merging**